pkg/plugins/transport/http_test.go

```diff
@@ -0,0 +1,20 @@
+package transport
+
+import (
+	"io"
+	"net/http"
+	"testing"
+)
+
+func TestNewHTTPTransport(t *testing.T) {
```

Comment:
nit: `NewHTTPTransport` is just the constructor. I think it would be more appropriate to call this `TestHTTPTransport`.